amgut/db/patches/0029.sql

```diff
@@ -0,0 +1,121 @@
+-- October 12, 2015
```

Comment:
Just a general comment - is this dropping information w/o moving it to the new structure? Doesn't it looks dangerous?